windows/src/developer/kmcmpdll/kmcmpdll.vcxproj

```diff
@@ -66,11 +66,10 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup>
     <_ProjectFileVersion>10.0.30319.1</_ProjectFileVersion>
-    <OutDir Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">$(SolutionDir)$(Platform)\$(Configuration)\</OutDir>
+    <OutDir Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">$(Platform)\$(Configuration)\</OutDir>
```

Comment:
Spotted this while debugging -- when loading the developer group solution this would end up with files in the wrong place.